lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.6.1-SNAPSHOT",
+  "version": "2.6.3-SNAPSHOT",
```

Comment:
will this change the numbers back to 2.6.3-snapshot?
if so, should we just update this to 2.7.0-snapshot?